gmso/core/angle.py

```diff
@@ -26,12 +27,24 @@ class Angle(Connection):
 
     __members_creator__: ClassVar[Callable] = Atom.model_validate
 
+    connectivity: ClassVar[Tuple[Tuple[int]]] = ((0, 1), (1, 2))
```

Comment:
We should add a description here.

Reply:
Also, do we need an alias?

Reply:
I specified connectivity as a ClassVar, so it's not modifiable across classes, and doesn't get treated the same way as other pydantic variables, which are defined under with pydantic Field.

Reply:
So also no alias here.